package/toltec-bootstrap/package

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env bash
+# Copyright (c) 2020 The Toltec Contributors
+# SPDX-License-Identifier: MIT
+
+pkgnames=(toltec-bootstrap)
+pkgdesc="Toltec installation script"
+url=https://toltec-dev.org/
+pkgver=0.0.1-1
+timestamp=2020-12-27T18:48Z
+section=utils
+devices=(rm1 rm2)
```

Comment:
```suggestion
devices=(any)
```

Reply:
You can't install the current bootstrap without this, so I think it's fair for it to be marked as any.

Reply:
I would say it should stay non-any since it'll probably do specific tests based on the ...soc0/mashine file and compare it against known version. So it would need an explicit update and therefore not classify for `any`.

Reply:
Since you use the bootstrap script to install, it by definition works on any device. It may refuse to continue the install on an unknown device, but the script itself should fully support any device out there.